capstone-ui/src/components/App/App.jsx

```diff
@@ -121,6 +157,53 @@ function App() {
         }
       });
   }
+
+  const saveInterests = () => {
+    setIsLoading(true);
+    axios
+      .post(`${config.API_BASE_URL}/profile/interests`, {
+        interests: {
+          skill: selectedSkill
+            ? selectedSkill.value
+            : newSkill
+            ? newSkill
+            : null,
+        },
+      })
+      .then(function () {
+        getInterests();
+        setIsLoading(false);
+        navigate("/profile/interests");
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  };
+
+  function addSkill(category, index) {
+    if (document.getElementById("add-skill")) {
+      setNewSkill({
+        name: document.getElementById("add-skill").value,
```

Comment:
Ideally name comes from a parameter in the function, just as category, and index.
With React, we prefer avoid using the DOM directly.